index.js

```diff
@@ -65,6 +65,8 @@ module.exports = (archive, dir, opts, cb) => {
         entry = entries[hyperPath] = entry || {}
         entry.length = stat.size
         entry.mtime = stat.mtime.getTime()
+        status.fileCount++
```

Comment:
if you check `mode` here and if `mode === 'updated'` change totalSize and fileCount as below it might work